examples/image-test-workflow/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: argo-hub.image-test-template.0.0.1
+  annotations:
+    argo-hub/version: '0.0.1'
+    argo-hub/description: 'Workflow template for a deployment that builds and tests an image through service containers'
+    argo-hub/categories: 'argo'
+    argo-hub/license: 'MIT'
+    argo-hub/owner_name: 'Matthew Chung'
+    argo-hub/owner_email: 'matthew.chung@codefresh.io'
+    argo-hub/owner_avatar: 'https://github.com/matthewchungcodefresh.png'
+    argo-hub/owner_url: 'https://github.com/matthewchungcodefresh'
+    argo-hub/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/image-test-workflow/assets/icon.svg"
+    argo-hub/icon_background: "#f4f4f4"
+spec:
+  volumes:
+    - name: docker-config
+      secret:
+        items:
+          - key: .dockerconfigjson
+            path: config.json
+        secretName: '{{ inputs.parameters.DOCKER_CONFIG_SECRET }}'
+  entrypoint: main
+  templates:
+    - name: main
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Builds a docker image using Kaniko and tests the image through service containers'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/image-test-workflow/assets/icon.svg"
```

Comment:
need to be change to this "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/image-test-workflow/assets/icon.svg"